packages/api/config/api.yml

```diff
@@ -6,6 +6,7 @@ ApiEndpoints:
   source: 'node_modules/@cumulus/api/dist/'
   apiRole: true
   urs: true
+  launchInVpc: true
```

Comment:
Do we also need to update the distribution API lambda in `packages/api/config/distribution.yml` to launch in the VPC?

Reply:
No, we shouldn't because it doesn't use elasticsearch, right?

Reply:
Ah, I thought it was required that all lambdas have to run in the VPC. But no, it doesn't use Elasticsearch